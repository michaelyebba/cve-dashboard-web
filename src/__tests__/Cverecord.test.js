import React from 'react';
import {HashRouter} from 'react-router-dom'
import renderer from 'react-test-renderer';
import CveRecord from '../components/datagrid/Cverecord';

it('renders correctly', () => {

  let cve = {
    id: "CVE-1234-456"
  }

  const tree = renderer
    .create(
        <HashRouter>
            <CveRecord cve={cve} />
        </HashRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});