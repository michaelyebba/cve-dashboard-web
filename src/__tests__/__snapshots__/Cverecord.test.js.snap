// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<tr>
  <td>
    <a
      href="#/cve/CVE-1234-456"
      onClick={[Function]}
    >
      CVE-1234-456
    </a>
  </td>
  <td />
  <td />
  <td />
  <td />
</tr>
`;
